collector.go

```diff
@@ -220,6 +228,26 @@ func getDatabaseIndexes(dbStats *dbStats) float64 {
 	return value
 }
 
+func countDatabaseTasksByStatus(data []byte, status string) int {
+	count := 0
+	jp.ArrayEach(data, func(value []byte, dataType jp.ValueType, offset int, err error) {
+		if taskConnectionStatus, _ := jp.GetString(value, "TaskConnectionStatus"); taskConnectionStatus == status {
+			count++
+		}
+	}, "OngoingTasksList")
```

Comment:
Is this correct? I'm testing with RavenDB 6.0.2 and the collection seems to be named `OngoingTasks`.

Reply:
This is correct in RavenDB 5.4.110. I'll check if we can support both